climada/hazard/test/test_tc_tracks.py

```diff
@@ -119,23 +119,73 @@ def test_read_raw_pass(self):
         self.assertEqual(track_ds.sid, '2017242N16333')
         self.assertEqual(track_ds.name, 'IRMA')
         self.assertEqual(track_ds.orig_event_flag, True)
-        self.assertEqual(track_ds.data_provider, 'ibtracs')
+        self.assertEqual(track_ds.data_provider, 'ibtracs_official_3h_mixed')
```

Comment:
I decided to rename the provider field to be more specific about the mixing of agencies involved.